frontend/src/components/ProfileBubble.jsx

```diff
@@ -6,25 +6,19 @@ import PropTypes from 'prop-types';
 
 class ProfileBubble extends Component {	
 
-	constructor(props) {
-		super(props);
-		this.state = {
-		}
-	}	
-
 	render() {
-		
-		//TODO: Make these link to profile using a provided UUID
+		let $userID = this.props.userID.split('/').pop();
+	
 		if (this.props.profilePicture === "$No profile picture provided") {
 			return(
-				  	<Link to="/profile" className={this.props.profileBubbleClassAttributes}>
+				  	<Link to={$userID} className={this.props.profileBubbleClassAttributes}>
```

Comment:
according to the example-article.json
![image](https://user-images.githubusercontent.com/25044820/54482756-26d8df80-480e-11e9-9280-b21585730c63.png)

would it be better to have the userID to be in this form to ensure consistency when dealing with authors not from out server